services/kaptain/metadata.yaml

```diff
@@ -1,7 +1,7 @@
 displayName: Kaptain
 description: Enterprise-grade ML/AI platform built on pure upstream KubeFlow
 category:
-  - AI/ML
+  - aiml
```

Comment:
I believe that this value is displayed in the UI. "AI/ML" is the value that @chessahoekstra and the UI team want to display. See https://docs.google.com/spreadsheets/d/1C8KDQZNdTp5ewxl_YrvDtoFGz9f3_3-y4yIeyY_cneA/edit#gid=0

Reply:
Please check [this thread](https://mesosphere.slack.com/archives/CQZ44D4SG/p1649164454879139)